### PR TITLE
feat(fetch-engine, get-platform): linux-arm support

### LINKED ISSUE #4573: Integrate Linux ARM in TypeScript

### PR DESCRIPTION
**Pull request by williamluke4**
closes #4573

### CODE REVIEW

**Comment by williamluke4** [comment]
Tested on an ec2 instance and is working

File: src/packages/get-platform/src/getPlatform.ts [hunk 1]
```diff
@@ -176,6 +193,9 @@ export async function getPlatform(): Promise<Platform> {
   if (platform === 'netbsd') {
     return 'netbsd'
   }
+  if (platform === 'linux' && arch === 'arm64') {
```
Comment (Jolg42) [inline src/packages/get-platform/src/getPlatform.ts:196]: We will support only arm64 or arm too (32 I guess?)

Comment (williamluke4) [reply src/packages/get-platform/src/getPlatform.ts:196]: Currently only `arm64`

**Review by timsuchanek — APPROVED** [review]
LGTM

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**